### ISSUE TITLE
To anyone asking why the cheat isn't working first the new upd broke alot of sigs + the are offsets in cheat.

### ISSUE THREAD

**Issue by dankor33** [T1GxR/Axion-CS2-RAGE-CHEAT#11]
its based of asphyxia so if you want to fix it just check out the repo but still it won't fix everything.

[CS2 Internal aka asphyxia ](https://github.com/maecry/asphyxia-cs2/tree/master)

**Comment by T1GxR** [comment]
That's true, especially the schemas are imported from asphyxia which can be helpful in updating the cheat!

**Comment by AdombSource** [comment]
@T1GxR will you update it to the new cs2 update?

**Comment by dankor33** [comment]
> @T1GxR will you update it to the new cs2 update?

he prob won't source fix it meh better than anything I can code but still if you want badly upd just copy upds from asphyxia then fix then broken sigs in ida.

**Comment by AdombSource** [comment]
> > @T1GxR will you update it to the new cs2 update?
> 
> he prob won't source fix it meh better than anything I can code but still if you want badly upd just copy upds from asphyxia then fix then broken sigs in ida.

I already tried to do that, but the problem is, that the source is not the same as the build in releases tab, also when I copy the updated lines from asphyxia I get like 2000 build errors from only 2 lines of plus code lmaoo

**Comment by AdombSource** [comment]
might be a me problem cuz I basically know nothing about cs2 cheat coding. I know how to use visual studi, aswell as a little bit of c++ but thats it, basically im an idiot trying to paste an update from the base to this

**Comment by T1GxR** [comment]
> > > @T1GxR will you update it to the new cs2 update?
> > 
> > 
> > he prob won't source fix it meh better than anything I can code but still if you want badly upd just copy upds from asphyxia then fix then broken sigs in ida.
> 
> I already tried to do that, but the problem is, that the source is not the same as the build in releases tab, also when I copy the updated lines from asphyxia I get like 2000 build errors from only 2 lines of plus code lmaoo

It is the same but you are just idiot. First of all, you have to build it on publish and not on release/debug then update everything inside. 
I'm currently working for other projects and i don't have time to update this one, at least these days. I'll try my best to update it as soon as possible!

**Comment by AdombSource** [comment]
> > > > @T1GxR will you update it to the new cs2 update?
> > > 
> > > 
> > > he prob won't source fix it meh better than anything I can code but still if you want badly upd just copy upds from asphyxia then fix then broken sigs in ida.
> > 
> > 
> > I already tried to do that, but the problem is, that the source is not the same as the build in releases tab, also when I copy the updated lines from asphyxia I get like 2000 build errors from only 2 lines of plus code lmaoo
> 
> The source is the same but you are just idiot. First of all, you have to build it on publish and not on release/debug then update everything inside. (That's why you get over 2000 errors, because its not on publish). I'm currently working for other projects and i don't have time to update this one, at least these days. I'll try my best to update it as soon as possible!

Lmao okay i will try to do it tomorrow, thanks(probably still wont be able to do it since im retarded)

**Comment by T1GxR** [comment]
> > > > > @T1GxR will you update it to the new cs2 update?
> > > > 
> > > > 
> > > > he prob won't source fix it meh better than anything I can code but still if you want badly upd just copy upds from asphyxia then fix then broken sigs in ida.
> > > 
> > > 
> > > I already tried to do that, but the problem is, that the source is not the same as the build in releases tab, also when I copy the updated lines from asphyxia I get like 2000 build errors from only 2 lines of plus code lmaoo
> > 
> > 
> > The source is the same but you are just idiot. First of all, you have to build it on publish and not on release/debug then update everything inside. (That's why you get over 2000 errors, because its not on publish). I'm currently working for other projects and i don't have time to update this one, at least these days. I'll try my best to update it as soon as possible!
> 
> Lmao okay i will try to do it tomorrow, thanks(probably still wont be able to do it since im retarded)

The hardest and most annoying part of this source code is how signatures were made, it has asphyxia base but it was made terrible and it's hard to understand how it works, especially when all signaturs are imported in CS_XOR.

**Comment by MattPlayGamez** [comment]
Did anyone find a solution?
